pkg/accountmanager/service.go

```diff
@@ -86,6 +86,42 @@ func (s *Service) UpsertPrincipalAccess(account *account.Account) error {
 	return nil
 }
 
+// DeletePrincipalAccess removes all the principal roles and policies
+func (s *Service) DeletePrincipalAccess(account *account.Account) error {
```

Comment:
Is this new behavior?

Reply:
Moved code from role and policy manager.